MIGRATION_GUIDE_v3_to_v5.md

```diff
@@ -165,7 +173,10 @@ The user name space is accessible via `OneSignal.User` and provides access to us
 | ----------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `IPushSubscription PushSubscription`                  | *The push subscription associated to the current user.*                                                                                                                                                                                  |
 | `string Language`                                     | *Set the 2-character language either as a detected language or explicitly set for this user.*                                                                                                                                            |
```

Comment:
Can you also add the OneSignalId and ExternalId properties in the table as well?